src/main/java/com/minecolonies/coremod/blocks/interfaces/IBlockMinecolonies.java

```diff
@@ -1,6 +1,7 @@
-package com.minecolonies.coremod.blocks;
+package com.minecolonies.coremod.blocks.interfaces;
 
 import net.minecraft.block.Block;
+import net.minecraft.block.state.IBlockState;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'net.minecraft.block.state.IBlockState'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)